docs/apm/apm-alerts.asciidoc

```diff
@@ -63,8 +67,8 @@ This guide creates an alert for the `opbeans-python` service based on the follow
 From the APM app, navigate to the `opbeans-python` service and select
 **Alerts** > **Create threshold alert** > **Error rate**.
 
-The name of your alert will automatically be set as `Error rate | opbeans-python`,
-and the alert will be tagged with `apm` and `service.name:opbeans-python`.
+`Error rate | opbeans-python` is automatically set as the name of the alert,
+and `apm` and `service.name:opbeans-python` are added as tags.
 Feel free to edit either of these defaults.
```

Comment:
I'm late to this - but users should probably not edit the tags. We will likely use them as lookup mechanisms in the future, and they're enabled only because the Alerting UI does not offer a way for them to be disabled yet.

Reply:
Really good to know, thanks. I'll fix this.